src/UserLogin.ts

```diff
@@ -0,0 +1,21 @@
+import * as consts from "./Consts"
+import getClient from "./Client"
+
+export async function getUser(): Promise<string> {
+  const userIdFromLocalStorage: string | null = localStorage.getItem(
+    consts.brigadeUserIdKey
+  )
+
+  if (userIdFromLocalStorage) {
+    return userIdFromLocalStorage
+  }
+
+  try {
+    const fetchedUserId = (await getClient().authn().whoAmI()).id
+    localStorage.setItem(consts.brigadeUserIdKey, fetchedUserId)
+
+    return fetchedUserId
+  } catch {
+    return ""
```

Comment:
Returning no user ID seems ok for now. Eventually we'll figure out how to deal with this better.

Could you perhaps add a `// TODO: ...` note above the return so we don't lose track of the fact this needs some improvement?

Reply:
Okk Sure. Will add `// TODO: Improvement needed. Something better than empty string.`